helper.go

```diff
@@ -54,21 +54,15 @@ func Check(err error) {
 	}
 }
 
-func (x *GoSNMP) decodeValue(data []byte, msg string) (*variable, error) {
-	retVal := &variable{}
-
-	if len(msg) > 0 {
-		x.Logger.Printf("decodeValue: msg: %s", msg)
```

Comment:
Was it intended to drop this log message? Would rather keep it in, as we can now disable it with a build flag.

Reply:
Thanks for elaborating @it1804